packages/lit-element-starter-js/docs/_README/index.html

```diff
@@ -0,0 +1,4 @@
+<p>This directory containts the sources for the static site contained in the /docs/ directory. The site is based on the <a href="11ty.dev">eleventy</a> static site generator.</p>
```

Comment:
Hmm... I think the idea with the `_` prefix on _README.md was that it would be ignored by eleventy. Ideally we don't generate this.

Reply:
Done. Added an `.eleventyignore` to exclude.